modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -946,10 +983,32 @@ Status description:: error: syntax error or access rule violation - unsupported
 
 Status description:: error: syntax error or access rule violation - invalid parameter. Invalid parameter `{ $param }`.
 
+=== 42N53
+
+Status description:: error: syntax error or access rule violation - unsafe usage of repeatable elements. The quantified path pattern may yield an infinite number of rows under match mode 'REPEATABLE ELEMENTS` Use a path selector or add an upper bound to the quantified path pattern.
```

Comment:
Agree with @Lojjs. Also it turns out that, for now at least, we don't think it's safe to allow just path selectors under repeatable elements. So there is only one fix, adding an upper bound to any quantified path pattern. 

Here is my suggested change:

```suggestion
Status description:: error: syntax error or access rule violation - unsafe usage of repeatable elements. The quantified path pattern may yield an infinite number of rows under match mode 'REPEATABLE ELEMENTS'. Add an upper bound to the quantified path pattern.
```

Reply:
@renetapopova If you change it like this (also remove the path selector part) here in docs, I can fix it in the spreadsheet and codebase. I anyway plan to implement the last error code for planner today so can fix this in the same PR.

Reply:
Updated in the codebase here https://github.com/neo-technology/neo4j/pull/29116

Reply:
@NataliaIvakina FYI, this commented is still unhandled (or rather Reneta only fixed my complaint and not Finbar's)